sdk/storage/azure-storage-file-datalake/tests/perfstress_tests/read.py

```diff
@@ -0,0 +1,36 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure_devtools.perfstress_tests import get_random_bytes, WriteStream
+
+from ._test_base import _FileSystemTest
+
+
+class DownloadTest(_FileSystemTest):
```

Comment:
Extend `_FileTest`?

Reply:
`_FileTest` base currently creates a unique destination filename (and so is used for the upload tests).
So for the downloads tests I have only used the `_FileSystemTest` base and have each instance of the `DownloadTest` simply hit the same filename endpoint.

Reply:
(But I did have the `global_setup/cleanup` for `_FileTest` messed up - so have fixed that)

Reply:
There's really not much benefit to the `_FileTest` base - I could refactor it out

Reply:
Keep or remove `_FileTest` is up to you, it might be used more in the future, and .NET has a similar class, so my vote is to keep it.